src/main/java/com/dougnoel/sentinel/filemanagers/FileManager.java

```diff
@@ -98,4 +104,60 @@ public static String getClassPath(String className) {
 			return null;
 		}
 	}
+	
+	/**
+	 * Saves an image of type File to the given sub-directory of the configured, or default, image path.
+	 * 
+	 * @param subDirectory the sub-directory within the configured or default image path.
+	 * If the sub-directory is null, the root image directory will be used.
+	 * @param imageFileName the file name of the image to save.
+	 * @param imageFile the File of the image to save.
+	 */
+	public static void saveImage(String subDirectory, String imageFileName, File imageFile) throws IOException {
+		if(subDirectory == null) 
+			subDirectory = "";
+		else
+			subDirectory += "/";
```

Comment:
You repeat this code 3 times in three different methods. Move it into a separate method.

Reply:
I consolidated this and a bunch of other reused code to createImagePath which creates an File to use for saving images. There's only one method that needs a string, and getAbsolutePath solves it there.